miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/CachedMiniAppVerifier.kt

```diff
@@ -77,3 +71,17 @@ internal class CachedMiniAppVerifier
         private const val BUFFER_SIZE = 8192
     }
 }
+
+@Suppress("TooGenericExceptionCaught", "SwallowedException")
+private fun initEncryptedSharedPreference(context: Context) = try {
+    EncryptedSharedPreferences.create(
+        "com.rakuten.tech.mobile.miniapp.cache.hash",
+        MasterKeys.getOrCreate(MasterKeys.AES256_GCM_SPEC),
+        context,
+        EncryptedSharedPreferences.PrefKeyEncryptionScheme.AES256_SIV,
+        EncryptedSharedPreferences.PrefValueEncryptionScheme.AES256_GCM
+    )
+} catch (e: Exception) {
+    throw MiniAppSdkException("Something wrong with device keystore." +
```

Comment:
Maybe we should make a specific exception type for this which extends `MiniAppSdkException`? Maybe something like `MiniAppVerificationException`. What do you think?

Reply:
updated.